pkg/server/server_test.go

```diff
@@ -110,7 +109,7 @@ func startFakeProxy(t *testing.T) *http.Server {
 	mux := http.NewServeMux()
 	mux.HandleFunc("/proxyhealth", fakehealth)
 
-	srv := &http.Server{Addr: ":" + proxy.ProxyPort, Handler: mux, ReadHeaderTimeout: 2 * time.Second}
+	srv := &http.Server{Addr: ":8091", Handler: mux, ReadHeaderTimeout: 2 * time.Second}
```

Comment:
Let's add a comment here why we need to use a different port so we don't forget about this :)